src/error.rs

```diff
@@ -38,14 +43,34 @@ impl Error {
     pub fn code(&self) -> NonZeroU32 {
         self.0
     }
+    
+    fn msg(&self) -> Option<&'static str> {
+        if let Some(msg) = super::error_msg_inner(self.0) {
+            Some(msg)
+        } else {
+            match *self {
+                UNKNOWN => Some("getrandom: unknown error"),
+                UNAVAILABLE => Some("getrandom: unavailable"),
+                _ => None
+            }
+        }
+    }
+}
+
+impl fmt::Debug for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
+        match self.msg() {
+            Some(msg) => write!(f, "Error(\"{}\")", msg),
+            None => write!(f, "Error({})", self.0.get()),
```

Comment:
Maybe it's better to use `"Error({:08X})"` formatting here? Same for `Display` implementation. Also `nonZeroU32` already implements `Display`, so no need for `get()` here.